xml/hwbp_registers.xml

```diff
@@ -270,23 +285,86 @@
             by 1.
         </field>
 
-        <field name="action" bits="5:0" access="R/W" reset="0">
-            Determines what happens when this trigger matches.
-
-            0: Raise a breakpoint exception. (Used when software wants to use the
-            trigger module without an external debugger attached.)
+        <field name="action" bits="5:0" access="R/W" reset="0" />
+    </register>
 
-            1: Enter Debug Mode. (Only supported when \Fdmode is 1.)
+    <register name="Exception Trigger" short="etrigger" address="0x7a1">
+        This register is accessible as \Rtdataone when \Ftype is 4.
 
-            2: Start tracing.
+        If implemented, it allows triggering on a variety of interrupt and
+        exception sources. If such a trigger is taken, all the usual CSRs as
+        well as \Rpc are updated, and the requested action is taken just before
+        the first instruction of the interrupt/exception handler is executed.
 
-            3: Stop tracing.
+        The exceptions to trigger on are specified in \Rtdatatwo. Each of the
+        low 16 bits corresponds to the exceptions listed in the description of
+        \Rmcause in the Privileged Spec when the Exception bit is 0, while the
+        next 16 bits correspond to the exceptions listed there when the
+        Exception bit is 1. The full list is duplicated here in
+        Table~\ref{tab:etrigger}. An implementation may decide per bit whether
+        to implement it as R/W, or to tie it to 0.
 
-            4: Emit trace data for this match. If it is a data access match,
-            emit appropriate Load/Store Address/Data. If it is an instruction
-            execution, emit its PC.
+        \begin{table}[H]
+        \centering
+        \caption{\Retrigger exceptions in \Rtdatatwo}
+        \label{tab:etrigger}
+        \begin{tabular}{|r|l|}
+        \hline
+        Bit &amp; Description \\
+        \hline
+        0 &amp; Instruction address misaligned \\
```

Comment:
I don't think we should reproduce this here. We should just point to the relevant section of the ISA manual, and be clear which version of the ISA manual we're talking about

Reply:
And if we were keeping this, why isn't it in `Rtdatatwo` description? Why are we squeezing it into this register's description?